CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added DashboardItemOrder that allows to select course runs
 - Add make dbshell cmd to access database in cli
 
+### Fixed
+
+- course runs sync webhook when course has a snapshot
```

Comment:
Maybe I'm wrong, but the changelog doesn't clearly reflect this fix, imo.
Perhaps use the added test docstring instead ?